src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -650,9 +650,14 @@ object Helpers extends SparkSessionWrapper {
     val remoteConfig = remoteWorkspace.getConfig
     val etlDatabaseNameToCreate = if (localETLDatabaseName == "" & !usingExternalMetastore)  {remoteConfig.databaseName} else {localETLDatabaseName}
     val consumerDatabaseNameToCreate = 	if (localConsumerDatabaseName == "" & !usingExternalMetastore) {remoteConfig.consumerDatabaseName} else {localConsumerDatabaseName}
-    val LocalWorkSpaceID = if (dbutils.notebook.getContext.tags("orgId") == "0") {
-      dbutils.notebook.getContext.apiUrl.get.split("\\.")(0).split("/").last
-    } else dbutils.notebook.getContext.tags("orgId")
+//    val LocalWorkSpaceID = if (dbutils.notebook.getContext.tags("orgId") == "0") {
+//      dbutils.notebook.getContext.apiUrl.get.split("\\.")(0).split("/").last
+//    } else dbutils.notebook.getContext.tags("orgId")
+    val LocalWorkSpaceID = if (spark.conf.get("spark.databricks.clusterUsageTags.clusterOwnerOrgId") == " ") {
```

Comment:
I suggest to define a function that will be used everywhere instead of repeating the logic in the multiple places

Reply:
@alexott  Agreed and Done the changes